src/etc/inc/filter.lib.inc

```diff
@@ -94,9 +94,12 @@ function filter_core_get_antilockout()
         return [];
     }
 
-    $lockout_if = get_primary_interface_from_list(array_keys(legacy_config_get_interfaces(['virtual' => false])));
-    if (empty($lockout_if)) {
+    $lockout_ifs = array_keys(legacy_config_get_interfaces(['virtual' => false, 'enable' => true, 'antilockout' => true]));
+    if (empty($lockout_ifs)) {
```

Comment:
I'm still split on this fallback. It could also be considered to move `noantilockout` to only refer to the automatic anti-lockout behaviour and never mess with the interface-based setting.